include/powsybl/iidm/NetworkVariant.hpp

```diff
@@ -48,10 +56,19 @@ class VariantImpl : public Variant<Network, VariantImpl> {
 
     SynchronousComponentsManager& getSynchronousComponentsManager();
 
+private:
+    friend class network::BusView;
```

Comment:
This is needed for `BusBreakerView` to call `invalidateCache()` which is not const...